__tests__/test-utils/mock-data/manager.js

```diff
@@ -0,0 +1,398 @@
+      (version: FeedVersion): SummarizedFeedVersion => (
+        {
+          feedSource: version.feedSource,
+          id: version.id,
+          nextVersionId: version.nextVersionId,
+          previousVersionId: version.previousVersionId,
+          updated: version.updated,
+          validationResult: version.validationSummary,
+          version: version.version
+        }
+      )
+    ): Array<SummarizedFeedVersion>),
+    id: 'mock-deployment-id',
+    invalidFeedSources: [],
+    lastDeployed: null,
+    lastUpdated: 1553292345726,
+    name: 'mock-deployment',
+    organizationId: null,
+    osmFileId: null,
+    otpCommit: null,
+    project,
+    projectBounds: (null: ?Bounds), // not sure why this flow type annotation is needed (see https://git.io/fjJCm)
+    routerId: (null: ?string), // not sure why this flow type annotation is needed (see https://git.io/fjJCm)
+    user: (null: ?any) // not sure why this flow type annotation is needed (see https://git.io/fjJCm)
+  }
+}
+
+// a mock project without any feeds or deployments
+export const mockProject = {
+  autoFetchFeeds: true,
+  autoFetchHour: 0,
+  autoFetchMinute: 0,
+  bounds: null,
+  buildConfig: {
+    fares: null,
+    fetchElevationUS: null,
+    stationTransfers: null,
+    subwayAccessTime: null
+  },
+  dateCreated: 1526305321824,
+  defaultTimeZone: null,
+  deployments: [],
+  feedSources: [],
+  id: 'mock-project-id',
+  lastUpdated: 1553236399556,
+  name: 'mock-project',
+  organizationId: null,
+  otpServers: [],
+  pinnedDeploymentId: null,
+  routerConfig: {
+    carDropoffTime: null,
+    numItineraries: null,
+    requestLogFile: null,
+    stairsReluctance: null,
+    updaters: null,
+    walkSpeed: null
+  },
+  useCustomOsmBounds: false,
+  user: null
+}
+
+// a mock feed with a version
+export const mockFeedWithVersion = {
+  dateCreated: 1536195264564,
+  deployable: false,
+  editorNamespace: 'gzek_ztquwiwjmselpeswgbsrre',
+  externalProperties: {},
+  id: 'mock-feed-with-version-id',
+  isPublic: false,
+  lastFetched: 1543389038810,
+  lastUpdated: 1543389038810,
+  latestValidation: {
+    agencies: null,
+    agencyCount: 1,
+    avgDailyRevenueTime: 0,
+    bounds: {
+      north: 39.0486949672717,
+      south: 38.92884,
+      east: -76.481211,
+      west: -76.5673055566884
+    },
+    endDate: '20190801',
+    errorCount: 78,
+    loadFailureReason: null,
+    loadStatus: 'SUCCESS',
+    routeCount: 10,
+    startDate: '20180801',
+    stopCount: 237,
+    stopTimesCount: 11170,
+    tripCount: 415
+  },
+  latestVersionId: 'mock-feed-version-id',
+  name: 'test feed with a version',
+  noteCount: 0,
+  organizationId: null,
+  projectId: mockProject.id,
+  publishedVersionId: null,
+  retrievalMethod: 'MANUALLY_UPLOADED',
+  s3Url: null,
+  snapshotVersion: null,
+  url: 'http://mdtrip.org/googletransit/AnnapolisTransit/google_transit.zip',
+  user: null
+}
+
+// a mock feed with no versions
+export const mockFeedWithoutVersion = {
```

Comment:
It seems like this should have a type defined. Is there a reason why it doesn't?

Reply:
The typing of these mock objects are checked when they are sent to various methods as function arguments. If the object structure doesn't match the argument type, a flow error occurs.